fixed_EE_trafo: [1., 0., 0., 0, 0., 1., 0., 0., 0., 0., 1., 0, 0, 0, 0.1034, 1]

# 1 0 0 0
# 0 1 0 0
# 0 0 1 0.1034
# 0 0 0 1 